charts/index.yaml

```diff
@@ -3,7 +3,25 @@ entries:
   azuredisk-csi-driver:
   - apiVersion: v1
     appVersion: latest-v2
-    created: "2022-08-09T21:58:39.780908374-07:00"
+    created: "2022-08-11T14:09:09.548595076-07:00"
+    description: Azure disk Container Storage Interface (CSI) Storage Plugin
+    digest: a00620fe9480b785d05a692c997f6d4964cf99ede7ed333f39bf81e6b2d4518a
+    name: azuredisk-csi-driver
+    urls:
+    - https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/main_v2/charts/latest-v2/azuredisk-csi-driver-v2.0.0-beta.7.tgz
+    version: v2.0.0-beta.7
+  - apiVersion: v1
+    appVersion: v2.0.0-beta.6
+    created: "2022-08-11T14:09:09.585067103-07:00"
+    description: Azure disk Container Storage Interface (CSI) Storage Plugin
+    digest: 0e7280d2bcd752668c4439078ffdfc4567ac97af97d8baa07c322b99c34fb741
+    name: azuredisk-csi-driver
+    urls:
+    - https://raw.githubusercontent.com/kubernetes-sigs/azuredisk-csi-driver/main_v2/charts/v2.0.0-beta.6/azuredisk-csi-driver-v2.0.0-beta.6.tgz
+    version: v2.0.0-beta.6
+  - apiVersion: v1
+    appVersion: latest-v2
```

Comment:
There's a duplicate entry for `latest-v2`. Re-index now that the beta.6 tgz has been deleted?